VisionDemo/projects/VisionDemo.json

```diff
@@ -0,0 +1,245 @@
 access tokens. You will then need to create a new access token for this namespace. \n\n4: Now, you will need to create a directory that holds the necessary files (as described here: https://internal.vantiq.com/docs/system/cli/index.html#load). I wanted to include a way to use multiple different models, so I included coco-1.2, yolov3-1.0, and yolo_open_image-1.0, which I defaulted to calling yolo-1.0. You will need to download two files and then create the third within a directory corresponding to each model.\n\nHere are the coco files:\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/coco/1.2/coco-1.2.meta\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/coco/1.2/coco-1.2.pb\nFor the .json file, adding these line should suffice:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"coco-1.2\"\n}\n\nHere are the yolov3 files:\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/yolov3/1.0/yolov3-1.0.meta\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/yolov3/1.0/yolov3-1.0.pb\nFor the .json file, adding these line should suffice:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"yolov3-1.0\"\n}\n\nHere are the yolo open image files:\nhttps://vantiqmaven.s3-us-west-2.amazonaws.com/vantiq/models/yolo_open_image/1.0/yolo_open_image-1.0.meta\nhttps://vantiqmaven.s3-us-west-2.amazonaws.com/vantiq/models/yolo_open_image/1.0/yolo_open_image-1.0.pb\nFor the json file:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"yolo-1.0\"\n}\n\n5: Once you have those three files in a directory corresponding to the specifc model, you can finally load the model to the namespace using a command similar to this:\nvantiq load tensorflowmodel coco-1.2 -t (access token) -b https://internal.vantiq.com -n vision-demo\n\n\nOnce you have finished these steps, you are all set to run the demo. However, if you would like to use your own images as part of the TensorFlow demo, you can download images to your computer and then upload using the client button. If you would like to use yor own images as part of the VisionScript demo, follow these steps.\n\n\n1: Load an image of your choosing through the vantiq CLI. Use an image with people's faces as this part of the demo finds people's faces in an image. You will use a command along these lines: \nvantiq load image (directory containing image name) -t (access token) -b https://internal.vantiq.com -n vision-demo\n\n2: Once you have loaded your image, you can go and check to make sure it is there by going to show, find records, check show system types, and select system.images. \n\n3: Once you see the image you have uploaded, you will need to edit the vsExample procedure to use your image name instead of blood.jpg on line 33 in the processImage method call and on line 21 in the VSPage On Start function. \n\n\nAdditional Documentation:\nVisionScript: https://dev.vantiq.com/docs/system/rules/index.html#VisionScriptBuilder\n\nTensorFlow: \nhttps://dev.vantiq.com/docs/system/imageprocessing/index.html#image-analysis",
```

Comment:
couple typos in the description here:  
     specifc  -> specific
     yor       -> your
Also, some URLs reference dev.vantiq.com & some use internal.vantiq.com.  They should all be the same, so you should change internal to dev.vantiq.com